app/views/katello/api/v2/alternate_content_sources/base.json.rabl

```diff
@@ -2,18 +2,23 @@ object @resource
 
 extends 'katello/api/v2/common/identifier'
 
-attributes :name, :base_url, :subpaths, :content_type, :alternate_content_source_type, :smart_proxies, :upstream_username, :verify_ssl
-
-child :ssl_ca_cert => :ssl_ca_cert do |_object|
-  attributes :id, :name
-end
+if @resource.respond_to?(:alternate_content_source_type)
```

Comment:
@resource is null here for some reason so the index doesn't return all expected field. One workaround I tested locally is to not use object @resource and reference @object instead of @resource in the file.